### ISSUE TITLE
PP: 0.76 Compact World Update

### ISSUE THREAD

**Issue by NotAFile** [piqueserver/aosprotocol#1]
The .76 world update is much more compact than the .75 world update.

It might be worthwhile backporting it to .75 OS under a new packet ID

**Comment by xtreme8000** [comment]
While it is more compact, there is one better solution:
`uint32_t[] - bitmask array, every bit equals one player id`

**Comment by NotAFile** [comment]
I'm not quite sure what you mean? If I understand it correctly, your format just saves one byte for the ID per player?

**Comment by xtreme8000** [comment]
yes

**Comment by NotAFile** [comment]
True, at 10ups 32 players clients just save around 240B/s which is not really significant, but the server saves 8KB/s. This figure will only grow with 64 players and higher network ups so it would probably make sense.

**Comment by xtreme8000** [comment]
I mean if we want to optimise bandwith, then do it at the most possible level. Especially considering protocol updates are hard to force on players.
